src/views/Viewer.vue

```diff
@@ -48,6 +48,12 @@
 				@click="showSidebar">
 				{{ t('viewer', 'Open sidebar') }}
 			</ActionButton>
+			<ActionButton
+				v-if="Sidebar && !isSidebarShown"
+				icon="icon-delete-white"
```

Comment:
Should be black, or the icon is not shown on top of white background :)
```suggestion
				icon="icon-delete"
```

Reply:
Same for `icon-menu-sidebar-white-forced`

Reply:
I'd suggest you create a computed named `canDelete` or something :)

Reply:
https://github.com/nextcloud/server/blob/6f8147b18fadd749b6060ab540292c9ac6f85d61/apps/dav/lib/Connector/Sabre/FilesPlugin.php#L316-L323
https://github.com/nextcloud/server/blob/89ed2c37bf656ceb772bb6759c8977a7dc78b3fb/apps/dav/lib/Connector/Sabre/Node.php#L338-L368

Reply:
> 
> 
> Should be black, or the icon is not shown on top of white background :)

Changed it becaus of Jans suggestion in the Photos team public chat: 
"- Will it work with actual video previews? Or should we maybe make it icon-video-white-shadow for that case?"

Reply:
> 
> 
> Same for `icon-menu-sidebar-white-forced`, If there is only one icon shown, it needs to stay white, but if both the open sidebar AND the delete button are shown, then you need to use the standard icon `icon-menu-sidebar`

This is not my code

Reply:
No this is not, but you're changing things next to it, so it needs to be adapted

Reply:
> Changed it becaus of Jans suggestion in the Photos team public chat:

Well, currently it doesn't show any icon as the menu background is white :wink: